opds.py

```diff
@@ -407,24 +410,27 @@ class AcquisitionFeed(OPDSFeed):
     FEED_CACHE_TIME = int(Configuration.get('default_feed_cache_time', 600))
 
     @classmethod
-    def groups(cls, _db, title, url, lane, annotator, cache_type=None,
-               force_refresh=False, use_materialized_works=True):
+    def groups(cls, _db, title, url, lane, annotator, use_cache=True,
+               cache_type=None, force_refresh=False, use_materialized_works=True):
```

Comment:
I'm not totally against this, but rather than adding a new argument to this method I think it would be better to add to the behavior of cache_type.

Either make the default value of cache_type be the same as CachedFeed.GROUPS_TYPE, or (more reliable) define a constant `NO_CACHE = object()` which you can pass in to turn off caching.